app/assets/stylesheets/embed.css.scss

```diff
@@ -21,7 +21,6 @@ body {
     width: 100%;
     height: 100%;
     margin: 0;
-    overflow: hidden;
```

Comment:
Whats the default here? Usually it's `visible` unless you guys overwritten this somewhere? Assuming this should be `auto` or something if you want a scrollbar?

 I'll approve since I assume you tested this all and everything looks fine? ¯\_(ツ)_/¯

Reply:
Thanks for this. No default appears to be set. I've done another commit setting it to 'visible' (we want scrollbars, not surprises).